### ISSUE TITLE
[CCE] Problem to create node with Kubernetes 1.13.7

### ISSUE THREAD

**Issue by Gwenael22** [terraform-providers/terraform-provider-flexibleengine#228]
Hi there,

I have some problems to create nodes via Terraform on a cluster with version 1.13.7 of Kubernetes.

If i create a cluster and nodes in the CCE console. Everything is working fine.
If i create cluster and nodes via Terraform. The cluster is created but not the nodes. If I try to add a node in the cluster created via Terraform by using the console, the node is not created.

### Terraform Version
 v0.12.6

### Provider Version
 v1.6.0

### Affected Resource(s)
cce_nodes

### Error Output
Error: Error creating flexibleengine CCE Node: unexpected state 'Abnormal', wanted target 'Active'. last error: %!s(<nil>)

### Debug Output
2019/08/20 11:27:43 [DEBUG] flexibleengine_cce_node_v3.node_az_run[1]: apply errored, but we're indicating that via the Error pointer rather than returning it: Error creating flexibleengine CCE Node: unexpected state 'Abnormal', wanted target 'Active'. last error: %!s(<nil>)
2019/08/20 11:27:43 [TRACE] <root>: eval: *terraform.EvalMaybeTainted
2019/08/20 11:27:43 [TRACE] EvalMaybeTainted: flexibleengine_cce_node_v3.node_az_run[1] encountered an error during creation, so it is now marked as tainted
2019/08/20 11:27:43 [TRACE] <root>: eval: *terraform.EvalWriteState
2019/08/20 11:27:43 [TRACE] EvalWriteState: removing state object for flexibleengine_cce_node_v3.node_az_run[1]
2019/08/20 11:27:43 [TRACE] <root>: eval: *terraform.EvalApplyProvisioners
2019/08/20 11:27:43 [TRACE] EvalApplyProvisioners: flexibleengine_cce_node_v3.node_az_run[1] has no state, so skipping provisioners
2019/08/20 11:27:43 [TRACE] <root>: eval: *terraform.EvalMaybeTainted
2019/08/20 11:27:47 [DEBUG] flexibleengine_cce_node_v3.node_az_run[0]: apply errored, but we're indicating that via the Error pointer rather than returning it: Error creating flexibleengine CCE Node: Resource not found
2019/08/20 11:27:47 [TRACE] <root>: eval: *terraform.EvalMaybeTainted
2019/08/20 11:27:47 [TRACE] EvalMaybeTainted: flexibleengine_cce_node_v3.node_az_run[0] encountered an error during creation, so it is now marked as tainted
2019/08/20 11:27:47 [TRACE] <root>: eval: *terraform.EvalWriteState
2019/08/20 11:27:47 [TRACE] EvalWriteState: removing state object for flexibleengine_cce_node_v3.node_az_run[0]
2019/08/20 11:27:47 [TRACE] <root>: eval: *terraform.EvalApplyProvisioners
2019/08/20 11:27:47 [TRACE] EvalApplyProvisioners: flexibleengine_cce_node_v3.node_az_run[0] has no state, so skipping provisioners
2019/08/20 11:27:47 [TRACE] <root>: eval: *terraform.EvalMaybeTainted
2019/08/20 11:27:47 [TRACE] EvalMaybeTainted: flexibleengine_cce_node_v3.node_az_run[0] encountered an error during creation, so it is now marked as tainted
2019/08/20 11:27:47 [TRACE] <root>: eval: *terraform.EvalWriteState
2019/08/20 11:27:47 [ERROR] <root>: eval: *terraform.EvalApplyPost, err: Error creating flexibleengine CCE Node: Resource not found
2019/08/20 11:27:47 [ERROR] <root>: eval: *terraform.EvalSequence, err: Error creating flexibleengine CCE Node: Resource not found
2019/08/20 11:27:47 [TRACE] [walkApply] Exiting eval tree: flexibleengine_cce_node_v3.node_az_run[0]
2019/08/20 11:27:47 [TRACE] vertex "flexibleengine_cce_node_v3.node_az_run[0]": visit complete
2019/08/20 11:27:47 [TRACE] dag/walk: upstream of "provider.flexibleengine (close)" errored, so skipping
2019/08/20 11:27:47 [TRACE] dag/walk: upstream of "meta.count-boundary (EachMode fixup)" errored, so skipping
2019/08/20 11:27:47 [TRACE] dag/walk: upstream of "root" errored, so skipping

### Expected Behavior
Create nodes in the k8s cluster

### Actual Behavior
The node creation stop on an error and the CCE auto delete the node

### Steps to Reproduce
1. `terraform apply`

**Comment by niuzhenguo** [comment]
@Gwenael22 Can you provide your tf scripts?

**Comment by Gwenael22** [comment]
@niuzhenguo For information, I have added some details concerning the behavior in my ticket description

My scripts :
`resource "flexibleengine_cce_cluster_v3" "cluster_k8s" {
  name                   = "${lower(var.project_name)}-cluster"
  cluster_type           = "VirtualMachine"
  cluster_version        = "v1.13.7"
  flavor_id              = "cce.s2.small"
  vpc_id                 = local.vpc_id
  subnet_id              = flexibleengine_vpc_subnet_v1.gin_subnet.id
  container_network_type = "vpc-router"
}

resource "flexibleengine_cce_node_v3" "node_az_run" {
  count             = var.cce_number_of_run_nodes
  name              = "${lower(var.project_name)}-node-run-${format("%02d", count.index + 1)}"
  availability_zone = var.az_to_host_stack
  cluster_id        = flexibleengine_cce_cluster_v3.cluster_k8s.id

  flavor_id = "s3.xlarge.2"

  key_pair = flexibleengine_compute_keypair_v2.keypair.name

  root_volume {
    size       = 40
    volumetype = "SSD"
  }

  data_volumes {
    size = 500
    volumetype = "SSD"
  }
}
`

**Comment by niuzhenguo** [comment]
@Gwenael22 I tried with your scripts and got the same error. The cluster created but the node is Abnormal, then I tried to create node on console under the created cluster but got the same error.

**Comment by Gwenael22** [comment]
@niuzhenguo I think that the new version of CCE needs new parameter that is not documented for the moment ...

For information, i have also some problems with the new version 1.11.7 that is created without cluster roles ... The FE support informs me that there is a new parameters to add.

```
spec {
  ...
  "authentication" : {
    "mode": "rbac"
  }
}
```

**Comment by Gwenael22** [comment]
@niuzhenguo New CCE API documentation (new parameters)
[Cloud Container Engine API Reference 2.0.pdf](https://github.com/terraform-providers/terraform-provider-flexibleengine/files/3560317/Cloud.Container.Engine.API.Reference.2.0.pdf)

**Comment by niuzhenguo** [comment]
@Gwenael22 I can add authentication mode support, but seems that's not related to the error you raised.

**Comment by Gwenael22** [comment]
Hi @niuzhenguo , I agree with you but the new authentication mode make Terraform unusable to create CCE clusters. In fact, there is no default value for this parameter in CCE API ... for the moment (it's scheduled for a next deployment).

**Comment by Gwenael22** [comment]
@niuzhenguo after some tests today, the problem seems to be fixed concerning the version 1.13.7 of Kubernetes. The `authentication.mode = rbac` seems to be the default value in CCE API
